py-glaredb/src/lib.rs

```diff
@@ -86,6 +86,8 @@ fn connect(
     py: Python,
     data_dir_or_cloud_url: Option<String>,
     spill_path: Option<String>,
+    disable_tls: Option<bool>,
+    cloud_addr: Option<String>,
```

Comment:
Ideally these (and spill path) would be kwargs. Otherwise this will get very unwieldy.

https://pyo3.rs/main/function/signature#using-pyo3signature--

Reply:
I'll make that improvement, thanks

Reply:
IMO I like what I got it. Do you see any benefit of `kwarg` over this?

```rust
#[pyo3(signature = (data_dir_or_cloud_url = None, /, *, spill_path = None, disable_tls = true, cloud_addr = String::from("https://console.glaredb.com")))]
fn connect(
    py: Python,
    data_dir_or_cloud_url: Option<String>,
    spill_path: Option<String>,
    disable_tls: bool,
    cloud_addr: String,
) -> PyResult<LocalSession> { }
```

- `/` defines the end of position args
- '*` defines the beginning of keyword-only args (it's a separate comma because you can't do `*arg=val`)

So this means:

- `data_dir_or_cloud_url` is always the first (positional) arg
- order after that doesn't matter
- all these extra opts have default values and are hard to misconfigure, because by the nature of using a keyword, you have to provide a value!

LMK if you disagree on this. My thinking is that if we did

```rust
#[pyo3(signature = (data_dir_or_cloud_url = None, /, **kwargs))]
fn connect(
    py: Python,
    data_dir_or_cloud_url: Option<String>,
    opts: Option<&PyDict>,
) -> PyResult<LocalSession> { }
```

There would be additional plumbing on `opts`

Reply:
Example use:

```py
import glaredb

cloud_url = "<user>:<pass>@<org>.remote.qa.glaredb.com:6443/<db_name>"

conn = glaredb.connect(cloud_url, cloud_addr="https://qa.glaredb.com", disable_tls=False)
```

Reply:
This looks reasonable to me, pinging @universalmind303 to see if he has opinions here.